cognee/context_global_variables.py

```diff
@@ -14,11 +16,43 @@
 graph_db_config = ContextVar("graph_db_config", default=None)
 session_user = ContextVar("session_user", default=None)
 
+vector_dbs_with_multi_user_support = ["lancedb"]
+graph_dbs_with_multi_user_support = ["kuzu"]
+
 
 async def set_session_user_context_variable(user):
     session_user.set(user)
 
 
+def check_multi_user_support():
```

Comment:
I'd recommend to rename this function to since it represent a flag/state. For example, `multi_user_support_available` or `multi_user_support_possible`